src/install_libs.py

```diff
@@ -151,10 +153,19 @@ def add_lib_path(self):
                     return
 
         path = get_rc_path()
-        
         path = os.path.expanduser(path)
+        shell = os.getenv("SHELL")
```

Comment:
Could you please initialize the `command` variable here, so LSPs/linters don't complain?
Essentially, just like that:

```suggestion
        shell = os.getenv("SHELL")
        
        command = None
        
```

Can you add

Reply:
done

Reply:
Done